--- tbb-2019_U9/build/linux.gcc.inc.orig	2019-12-02 19:29:10.832630387 +0100
+++ tbb-2019_U9/build/linux.gcc.inc	2019-12-02 19:31:27.408557158 +0100
@@ -66,10 +66,10 @@
 
 ifeq ($(cfg), release)
         # -g is set intentionally in the release mode. It should not affect performance.
-        CPLUS_FLAGS = -O2 -g
+        CPLUS_FLAGS = $(CXXFLAGS) -O2 -g
 endif
 ifeq ($(cfg), debug)
-        CPLUS_FLAGS = -DTBB_USE_DEBUG -O0 -g
+        CPLUS_FLAGS = $(CXXFLAGS) -DTBB_USE_DEBUG -O0 -g
 endif
 
 CPLUS_FLAGS += $(ITT_NOTIFY) -DUSE_PTHREAD -pthread
